gensim/models/poincare.py

```diff
@@ -1385,7 +1385,7 @@ def __init__(self, file_path, encoding='utf8', delimiter='\t'):
             Path to file containing a pair of nodes (a relation) per line, separated by `delimiter`.
             Since the relations are asymmetric, the order of `u` and `v` nodes in each pair matters.
             To express a "u is v" relation, the lines should take the form `u delimeter v`.
-            e.g: `kangaroo	mammal` is a tab-delimited line expressing a "`kangaroo is a mammal`" relation.
+            e.g: https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv
```

Comment:
I like the original example, it's clear and concrete, please keep it. You can add a link to a TSV too, that's fine.

The typo was `realtions` (gone now).

Reply:
@AMR-KELEG also, make this link clickable, use syntax
```
e.g: `gensim/test/test_data/poincare_hypernyms.tsv <https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv>`_.
```